model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -67,6 +71,7 @@ def __init__(
         infect_hosp_rate_rv: RandomVariable,
```

Comment:
These names could be made more clear.

- `infect_hosp_rate_rv` makes me think of something like a case-fataility-rate, but I think is actually the delay distribution?
- `hosp_report_prob_rv` is not more like an infection-hospitalization-ratio than a reporting probability, I think?

Reply:
And drop the `_rv` suffix?

Reply:
No, sorry. Keep the suffix.